handlers/mesh_ops_handlers.go

```diff
@@ -74,7 +74,8 @@ func (h *Handler) MeshAdapterConfigHandler(w http.ResponseWriter, req *http.Requ
 
 		logrus.Debugf("meshLocationURL: %s", meshLocationURL)
 		if strings.TrimSpace(meshLocationURL) == "" {
-			err := errors.New("meshLocationURL cannot be empty to add an adapter.")
+			err := errors.New("meshLocationURL cannot be empty to add an adapter")  //error strings should not be capitalized or end with punctuation or a newline
```

Comment:
@girishranganathan can we have the lint rules changed? I don't like this one.

Reply:
Assigning these PRs to @girishranganathan in order to identify whether golint is configurable and whether this rule can be changed or removed.